BookingsApi/BookingsApi/Controllers/JusticeUserController.cs

```diff
@@ -48,16 +48,17 @@ public async Task<IActionResult> GetJusticeUserByUsername(string username)
         }
         
         /// <summary>
-        /// Get list of Justice User.
+        /// Get list of Justice User filtered by passed term. If term is null then not filter applied.
```

Comment:
I think we may be able to come up with a better description here

Reply:
Please let me know what better description you have in mind

Reply:
Sorry I should have given an example.

How about:

```suggestion
        /// Get a list of justice users. Optionally provide a search term to filter for users that contain the given term in their first name, contact email or username.
```

Reply:
done